### ISSUE TITLE
Building the Submission System

### ISSUE THREAD

**Issue by justingosan** [appirio-tech/tc-common-tutorials#6]
Where are the files/folders needed for this located exactly?

**Comment by skyhit** [comment]
which challenge are you working on?

**Comment by justingosan** [comment]
This one https://www.topcoder.com/challenge-details/30054599/?type=develop&lc=

**Comment by skyhit** [comment]
you just need to check out the whole git repository, and navigate to submission-system folder, and follow the steps.

**Comment by justingosan** [comment]
yep that's what I'm doing. Do I skip this part then? https://github.com/appirio-tech/tc-common-tutorials/tree/master/submission-system#building-the-submission-system

**Comment by skyhit** [comment]
no, the source code will be attached in forum.
